buildkite/bazelci.py

```diff
@@ -1482,15 +1500,9 @@ def partition_targets(targets):
     return included_targets, excluded_targets
 
 
-def get_targets_for_shard(build_targets, test_targets, shard_id, shard_count):
+def get_targets_for_shard(test_targets, shard_id, shard_count):
     # TODO(fweikert): implement a more sophisticated algorithm
-    included_build_targets, excluded_build_targets = partition_targets(build_targets)
-    build_targets_for_this_shard = sorted(included_build_targets)[shard_id::shard_count] + [
-        "-" + x for x in excluded_build_targets
-    ]
-    test_targets_for_this_shard = sorted(test_targets)[shard_id::shard_count]
-
-    return build_targets_for_this_shard, test_targets_for_this_shard
+    return sorted(test_targets)[shard_id::shard_count]
```

Comment:
This is actually a problem with excluded targets (this bug existed already on the previous version).

Reply:
I fixed that some time ago - this method gets the expanded list of test targets as an input and then your sharding algo works fine :)